platform/cli/src/commands/enums/endPoints.js

```diff
@@ -0,0 +1,5 @@
+const endPoints = {
+  NPM_KEYWORD: 'https://registry.npmjs.com/-/v1/search?text=keywords:',
```

Comment:
Hmm, this is fine for now, but we should really check all registries in the .npmrc file.

I did this when checking for scoped dependencies here: https://github.com/OHIF/Viewers/blob/ohif-cli/platform/cli/src/commands/utils/validate.js#L62

But you'll have to search how to just get all of the used registiries. It should be a very common thing to do and realitvely easy.